### PR TITLE
docs: update tsconfig template link aio->adev

### PR DESCRIPTION
**Pull request by davidlj95**
## PR Checklist

Please check to confirm your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- **N/A** Tests for the changes have been added (for bug fixes / features)
- **N/A** Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
`tsconfig` template files point to a non-existent link in Angular.io . Also, the page no longer exists in Angular.dev

Issue Number: N/A

## What is the new behavior?
Point to Typescript official documentation about `tsconfig.json` + Angular compiler options

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Checkout #27713 for references to related PRs

### CODE REVIEW

File: packages/schematics/angular/web-worker/files/worker-tsconfig/tsconfig.worker.json.template [hunk 1]
```diff
@@ -1,4 +1,5 @@
-/* To learn more about this file see: https://angular.io/config/tsconfig. */
+/* To learn more about Typescript configuration file: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html. */
+/* To learn more about Angular compiler options: https://angular.dev/reference/configs/angular-compiler-options. */
```
Comment (alan-agius4) [inline packages/schematics/angular/web-worker/files/worker-tsconfig/tsconfig.worker.json.template:2]: Workers are not part of the Angular compilation. 

```suggestion
```

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Can you please update the commit message as per guidelines? 
https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#commit

Thanks

File: packages/schematics/angular/web-worker/files/worker-tsconfig/tsconfig.worker.json.template (hunk 1 above)
Comment (davidlj95) [reply packages/schematics/angular/web-worker/files/worker-tsconfig/tsconfig.worker.json.template]: Nice catch, done in c1ec6c1aae0ec56d9d9a250f624f66ac8c0bbf2c

**Comment by davidlj95** [comment]
> Can you please update the commit message as per guidelines? https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#commit

Sure! Done in c1ec6c1aae0ec56d9d9a250f624f66ac8c0bbf2c

> 
> Thanks

thx to you

**Review by alan-agius4 — APPROVED** [review]
LGTM thanks

**Pull request merged**